practices/managing-node-projects.md

```diff
@@ -135,3 +136,21 @@ Snyk not only provides alerts when a new vulnerability is detected in one of the
 [Using components with known vulnerabilities caused 24% of the top 50 breaches](https://snyk.io/blog/owasp-top-10-breaches/). Keeping dependencies up-to-date is therefore crucial to maintain our standards of security.
 
 Every node.js project must be added to the Springer Nature organisation in Snyk so it can be monitored for security vulnerabilities.
+
+## Handling dependencies from third parties
+
+A question that often comes up, should we commit the contents of `node_modules` directory or any selected module. The answer on balance is that we don't recommend it. 
+
+Wherever possible we suggest instead:
+
+i). For you to create a `package.json` file if not added already, where you can specify package names and appropriate version numbers.
+
+ii). For instances where you might need a specific package version or dependency, you can add in `package-lock` or `shrink-wrap` file. For instance in cases like
+- Our app requires dependency A which requires dependency B which requires dependency C which requires dependency D. There’s a vulnerability in package D, an updated is released. Using a package lock file will *prevent* the update from been used on a re-deploy, so our app will still be vulnerable, which is bad.
```

Comment:
This 
There’s a vulnerability in package D, an updated is released
should be 
There’s a vulnerability in package D, an update is released

This
*prevent* the update from been used
should be
*prevent* the update from being used

Reply:
@perryharlock -- thanks for spotting this, as this section came from a discussion on frontend channel this morning. Just proves the importance of proof reading everything.